lib/content-services/src/lib/tag/tags-creator/tags-creator.component.html

```diff
@@ -1,54 +1,38 @@
 <div class="adf-tags-creation">
+    <div *ngIf="tagNameControlVisible" class="adf-tag-name-field">
+        <div class="adf-tag-search-field">
+            <input #tagNameInput
+                    matInput
+                    autocomplete="off"
+                    [formControl]="tagNameControl"
+                    (keyup.enter)="addTag()"
+                    adf-auto-focus
+                    placeholder="{{'TAG.TAGS_CREATOR.TAG_SEARCH_PLACEHOLDER' | translate}}"
+            />
+        </div>
+        <mat-error *ngIf="tagNameControl.invalid && tagNameControl.touched">{{ tagNameErrorMessageKey | translate }}</mat-error>
+    </div>
     <p
        class="adf-no-tags-message"
-       *ngIf="!tags.length && !tagNameControlVisible">
+       *ngIf="showEmptyTagMessage">
         {{ 'TAG.TAGS_CREATOR.NO_TAGS_CREATED' | translate }}
     </p>
     <div
         class="adf-tags-list"
         [class.adf-tags-list-fixed]="!tagNameControlVisible"
         #tagsList>
-        <p
-            *ngFor="let tag of tags"
-            class="adf-tag adf-label-with-icon-button">
+        <span *ngFor="let tag of tags" class="adf-tag adf-label-with-icon-button">
             {{ tag }}
             <button
                 data-automation-id="remove-tag-button"
                 mat-icon-button
                 (click)="removeTag(tag)"
                 [attr.title]="'TAG.TAGS_CREATOR.TOOLTIPS.DELETE_TAG' | translate"
-                [disabled]="disabledTagsRemoving">
-                <mat-icon>remove</mat-icon>
+                [disabled]="disabledTagsRemoving"
+                class="adf-remove-tag">
+                <mat-icon>close</mat-icon>
             </button>
-        </p>
-    </div>
-    <div
-        class="adf-tag-name-field"
-        *ngIf="(!tagNameControlVisible && tags.length) || tagNameControlVisible"
-        [hidden]="!tagNameControlVisible">
-        <mat-form-field *ngIf="tagNameControlVisible">
-            <mat-icon matPrefix>search</mat-icon>
-            <mat-label id="adf-tag-name-input-label">
-                {{ 'TAG.TAGS_CREATOR.NAME' | translate }}
```

Comment:
TAG.TAGS_CREATOR.NAME is unused, can be removed from en.json file

Reply:
Done